content/src/content/jcr_root/apps/core/wcm/components/image/v1/image/_cq_dialog/.content.xml

```diff
@@ -59,6 +59,20 @@
                                 name="./linkURL"
                                 rootPath="/content"
                                 wrapperClass="cmp-image--editor-link"/>
+                            <linkTarget
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                fieldDescription="Open image link in a new tab."
+                                text="Open link in new tab"
+                                name="./linkTarget"
+                                value="true"
+                                wrapperClass="cmp-image__editor-link"/>
```

Comment:
what do we use this wrapper class for? I don't think it's needed.

Reply:
For hiding the property, when the isDecorative image checkbox is checked (be inline with functionality that is already in place), but it should be cmp-image--editor-link for the v1 image component...